Cargo.toml

```diff
@@ -326,26 +326,27 @@ tokio = { workspace = true, features = ["rt"] }
 # If you change this also change the versions in the the patch section for uv
 pep440_rs = { git = "https://github.com/astral-sh/uv", tag = "0.2.18" }
 pep508_rs = { git = "https://github.com/astral-sh/uv", tag = "0.2.18" }
-# deno_task_shell = { path = "../deno_task_shell" }
-#rattler = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_conda_types = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_digest = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_lock = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_networking = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_repodata_gateway = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_shell = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_solve = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_virtual_packages = { git = "https://github.com/mamba-org/rattler", branch = "main" }
-#rattler_conda_types = { path = "../rattler/crates/rattler_conda_types" }
-#rattler_digest = { path = "../rattler/crates/rattler_digest" }
-#rattler_networking = { path = "../rattler/crates/rattler_networking" }
-#rattler_repodata_gateway = { path = "../rattler/crates/rattler_repodata_gateway" }
-#rattler_shell = { path = "../rattler/crates/rattler_shell" }
-#rattler_solve = { path = "../rattler/crates/rattler_solve" }
-#rattler_virtual_packages = { path = "../rattler/crates/rattler_virtual_packages" }
-#rattler_lock = { path = "../rattler/crates/rattler_lock" }
-#rattler_package_streaming = { path = "../rattler/crates/rattler_package_streaming" }
-#rattler = { path = "../rattler/crates/rattler" }
+# # deno_task_shell = { path = "../deno_task_shell" }
+rattler = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_conda_types = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_digest = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_lock = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_networking = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_repodata_gateway = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_shell = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_solve = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
+rattler_virtual_packages = { git = "https://github.com/nichmor/rattler", branch = "feat/add-search-for-packages-for-repodata" }
```

Comment:
This can be removed right?

Reply:
not really - we need another rattler release for this